website/src/menu.tsx

```diff
@@ -16,20 +16,28 @@ import {
   drawerItem,
   drawerList,
   navButton,
+  navButtonChoices,
   navDrawer,
   navLink,
   navMenu,
+  subMenuItems,
 } from "./menu.css"
 import { closeBlock } from "./sprinkles.css"
 
-export const NavMenu = () => {
+export const NavMenu = (p: { menuID: number }) => {
   const location = useLocation()
-  const [{ data }] = Wordpress.useMainMenuQuery()
-  const menuItems = data?.menuItems?.nodes
+  const [{ data }] = Wordpress.useMenuByIdQuery({
+    variables: { slug: p.menuID },
+  })
+  const menus = data?.menus?.nodes
```

Comment:
Why did you need to query `menus` instead of `menuItems`?

Reply:
Querying menuItems seemed to just return items without regard to which menu they came from, and their parentID was usually null and so couldn't be filtered by that. Mean while querying menus made it easy to see all the nodes of a specific wordpress menu by filtering by ID. So, it seemed easier to query menus than menuItems.